indico/web/client/js/react/components/MathJax.jsx

```diff
@@ -0,0 +1,23 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+
+export default function MathJax({children}) {
+  const wrapperRef = useRef(null);
+
+  useEffect(() => {
+    window.mathJax(wrapperRef.current);
```

Comment:
can't current be `null` initially?

Reply:
Yep, but I haven't noticed any problems with that. However I think I will just make sure I don't pass `null` to `mathJax`